xcodeproj/internal/resources.bzl

```diff
@@ -343,6 +343,19 @@ def collect_resources(
         parent.dependency_paths.append(child_bundle_path)
 
     frozen_bundles = []
+    resource_to_owners = {}
+    for (resource, owner) in resource_info.owners.to_list():
+        # AppleResourceInfo treats resource files inside xcassets separately, whereas
+        # rules_xcodeproj only cares about .xcassets folder
+        if ".xcassets" in resource:
+            resource = resource.split(".xcassets")[0] + ".xcassets"
```

Comment:
You can use the `max_splits` argument here. Or maybe `partition` (like the code that normalizes it elsewhere, maybe just use `normalize_file_path`?).

Reply:
sounds good!